index.js

```diff
@@ -33,6 +33,14 @@ client.on('message', message => {
     const { content } = message
     const { username, discriminator } = message.author
 
+    if(message.guild) {
```

Comment:
Why are we checking for `guild` here? Don't we want to record every message, and just insert `null` if there is no guild?